src/Helpers/ContextHelper.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Helpers;
+
+use Illuminate\Support\Str;
+
+class ContextHelper
+{
+    public static function getHelper(
+        string $helper,
+    ) {
+        $helperClass = config("blt.helpers.$helper") ?? "Blumilk\BLT\Helpers\\" . Str::ucfirst($helper) . "Helper";
+
+        return new $helperClass();
```

Comment:
I think I would avoid things like this. Maybe `return match($helper) (...)` would be more safe?

Reply:
With cases like `"user" => new UserHelper()` and so on.

Reply:
Or maybe, just maybe, add something like `::getUserHelper()`?

Reply:
Something like this?